tests/test_template.py

```diff
@@ -38,3 +40,21 @@ def test_find_shapes_from_one_slide(template):
 def test_find_shapes_index_out_of_range(template):
     with pytest.raises(IndexError):
         shapes = template._find_shapes(0, 'logo')
+
+
+def test_save_pdf(template):
+    try:
+        subprocess.Popen(['libreoffice', '--version'],
```

Comment:
Would use the high-level `subprocess.run()` function.

Reply:
ok